src/libcore/panicking.rs

```diff
@@ -60,16 +89,18 @@ fn panic_bounds_check(location: &Location<'_>, index: usize, len: usize) -> ! {
         unsafe { super::intrinsics::abort() }
     }
 
-    panic_fmt(
+    panic_fmt_internal(
         format_args!("index out of bounds: the len is {} but the index is {}", len, index),
         location,
     )
 }
 
+// FIXME: remove this function on bootstrap bump, inline it into `panic_fmt` above
+// (which will be the only caller at that point).
```

Comment:
Hang on, can't you just add `#[cfg(bootstrap)]` on the location argument?
Could also work for `panic_bounds_check`.

I'd rather not have people move code around or rename things when bumping beta.

Reply:
> Hang on, can't you just add #[cfg(bootstrap)] on the location argument?
Could also work for panic_bounds_check.

Hm, I could try that. But will that also work inside the `panic!` macro (which calls `panic_fmt`)? I guess I could duplicate the macro.

> I'd rather not have people move code around or rename things when bumping beta.

Yeah, I agree. The alternative is to only do `panic_bounds_check` now, wait for a bump, and then we can do `panic_fmt` without issues.

Reply:
@eddyb what about this?